### DISCUSSION TITLE
FDS/firesimulation_sprinkler TABLE_DATA error

### DISCUSSION THREAD

**Discussion by saemibang** [firemodels/fds#11345] [Q&A]
An error is found as shown in the figure. However, I entered the TABLE DATA so that the sum of MASS FRACTION is 1. For what problem? I would appreciate it if you could let me know.
![image](https://user-images.githubusercontent.com/112543935/212457334-6be5c770-2145-4ab9-b8f8-afbccfda2c3d.png)

**Comment by drjfloyd** [comment]
The problem is likely what the error message says -- that the first row of TABLE1 has a bad input for mass flow (for example did you put a negative number when FDS expects a positive?). Please provide your inputs if you want more specific help.

**Reply by saemibang** [reply]
thank you, I entered the mass flow incorrectly!
